development-workflow.md

```diff
@@ -232,12 +232,12 @@ Follow the official documentation on [Creating and Managing Virtual Devices](htt
 
 #### Creating Android Virtual Device via command line tool
 
-The `avdmanager` is a tool that allows you to create and manage Android Virtual Devices (AVDs) from the command line. The `avdmanager` is provided in the Android SDK Tools package (25.3.0 and higher) and is located in `<ANDROID_HOME_PATH_HERE>/tools/bin/`. For more information about the avdmanager and how to use it to create AVDs, see the [official avdmanager documentation](https://developer.android.com/studio/command-line/avdmanager.html).
+The `avdmanager` is a tool that allows you to create and manage Android Virtual Devices (AVDs) from the command line. The `avdmanager` is provided in the Android SDK Tools package (25.3.0 and higher) and is located in `<ANDROID_HOME_PATH_HERE>/cmdline-tools/latest/bin/`. For more information about the avdmanager and how to use it to create AVDs, see the [official avdmanager documentation](https://developer.android.com/studio/command-line).
```

Comment:
```suggestion
The `avdmanager` is a tool that allows you to create and manage Android Virtual Devices (AVDs) from the command line. The `avdmanager` is provided in the Android SDK Tools package (25.3.0 and higher) and is located in `<ANDROID_HOME_PATH_HERE>/cmdline-tools/latest/bin/`. For more information about the avdmanager and how to use it to create AVDs, see the [official avdmanager documentation](https://developer.android.com/studio/command-line/avdmanager).
```

Reply:
@rigor789 It's fine by me either way, but the reason I removed `/avdmanager` from the URL is that it points to the old path in the filesystem

Reply:
@isapir both paths are still kind-of valid (it's a mess tbh), the old path is there by default, the "new" path is only there if you explicitly install cmdline tools. The real issue, and why I think it's a mess is that the old path does not work on java 11+ which is required for gradle7+...

Anyways, I changed it back because either way, the section of the docs talks about creating AVDs through avdmanager, so I feel like it's still relevant to link to that page specifically.